storage-providers/basics/README.md

```diff
@@ -34,6 +34,42 @@ You need to understand the various earning mechanisms in the Filecoin network.
 
 [Filecoin deals ->](../filecoin-deals/storage-deals.md)
 
+### Daily fees and startup readiness (FIP-0100)
+
+With the activation of [FIP-0100](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0100.md) in network version 25, all new sectors - and any sectors that are extended or updated - incur a daily fee.
+
+This fee replaces the previous batch fee model and introduces a predictable cost structure tied to each sector’s quality-adjusted power and the network’s circulating supply.
+
+The fee begins accruing the day after a sector is committed or extended. It is deducted automatically at the end of each proving deadline.
+
+The network first draws from vesting block rewards. If those are insufficient, it draws from the miner’s available balance. If both are empty, the unpaid amount becomes **fee debt**.
+
+Fee debt does not directly cause faults. However, it can impact operations:
+
+- A miner with fee debt may be blocked from submitting certain messages (e.g., pre-commits or recoveries).
+- If the balance is too low to pay for WindowPoSt messages, sectors may fault.
+
+To avoid this, storage providers should:
+
+- Keep a FIL buffer in the miner actor’s balance.
+- Avoid fully withdrawing unlocked funds unless upcoming rewards will cover future fees.
+
+### Startup considerations
+
+Miners become eligible to win block rewards once they reach **10 TiB of raw byte power (RBP)**.
+
+However, rewards are not guaranteed as soon as that threshold is met. Block production is probabilistic, and smaller miners may wait longer to win a block - especially when competing against larger ones.
+
+This creates a funding gap during the startup phase.
+
+New storage providers must plan for this by funding their miner actor with enough FIL to:
+
+- Cover daily fees during onboarding,
+- Support message submission (like WindowPoSt),
+- And continue sealing until rewards start arriving.
+
+While the amount of FIL required is relatively small compared to overall infrastructure costs, it is operationally critical. Without it, the miner may become stuck - unable to seal new sectors or submit required messages due to fee debt or insufficient balance.
+
```

Comment:
"unable to seal new sectors or submit required messages due to fee debt or insufficient balance" -> 
"unable to seal new sectors, submit required messages, and produce blocks and win block reward due to fee debt or insufficient balance."